docs/avdanos/contributing/Design Guidelines/website.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Website Design Ideas
+description: A couple of design ideas for the AvdanOS website.
+---
+
+import AutoPlay from "../../../../src/components/media/autoplay.js"
```

Comment:
please use `@site/src/components/media/autoplay.js`

Reply:
That was from my previous PR.